File: JobCandidatesFront/infinite-scroll-app/src/app/model/candidate.ts
/*export interface Candidate {
  firstAndLastName: string;
  JMBG: string;
  yearOfBirth: Date;
  mail: string;
  telephoneNumber: string;
  isEmployed: boolean;
  lastChange: Date;
}*/

interface CandidateInterface{
  firstAndLastName: string;
  JMBG: string;
  yearOfBirth: Date;
  mail: string;
  telephoneNumber: string;
  isEmployed: boolean;
  lastChange: Date;
}
export class Candidate implements CandidateInterface{
  firstAndLastName: string;
  JMBG: string;
  yearOfBirth: Date;
  mail: string;
  telephoneNumber: string;
  isEmployed: boolean;
  lastChange: Date;
  
  constructor(obj:CandidateInterface){
      this.firstAndLastName=obj.firstAndLastName;
      this.JMBG=obj.JMBG;
      this.yearOfBirth=obj.yearOfBirth;
      this.mail=obj.mail;
      this.telephoneNumber=obj.telephoneNumber;
      this.isEmployed=obj.isEmployed;
      this.lastChange=obj.lastChange;
  }
}

File: JobCandidates/src/main/java/com/example/jobcandidates/model/Candidate.java
package com.example.jobcandidates.model;

import jakarta.persistence.*;
import java.util.Date;

@Entity
public class Candidate {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column
    private String firstAndLastName;
    @Column
    private String JMBG;
    @Column
    private Date yearOfBirth;
    @Column
    private String mail;
    @Column
    private String telephoneNumber;
    @Column
    private Boolean isEmployed;
    @Column
    private Date lastChange;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getFirstAndLastName() {
        return firstAndLastName;
    }

    public void setFirstAndLastName(String firstAndLastName) {
        this.firstAndLastName = firstAndLastName;
    }

    public String getJMBG() {
        return JMBG;
    }

    public void setJMBG(String JMBG) {
        this.JMBG = JMBG;
    }

    public Date getYearOfBirth() {
        return yearOfBirth;
    }

    public void setYearOfBirth(Date yearOfBirth) {
        this.yearOfBirth = yearOfBirth;
    }

    public String getMail() {
        return mail;
    }

    public void setMail(String mail) {
        this.mail = mail;
    }

    public String getTelephoneNumber() {
        return telephoneNumber;
    }

    public void setTelephoneNumber(String telephoneNumber) {
        this.telephoneNumber = telephoneNumber;
    }

    public Boolean getEmployed() {
        return isEmployed;
    }

    public void setEmployed(Boolean employed) {
        isEmployed = employed;
    }

    public Date getLastChange() {
        return lastChange;
    }

    public void setLastChange(Date lastChange) {
        this.lastChange = lastChange;
    }
}